arch/x86/core/acpi.c

```diff
@@ -31,7 +31,7 @@ static bool check_sum(struct acpi_sdt *t)
 {
 	uint8_t sum = 0, *p = (uint8_t *)t;
```

Comment:
t? :-)

Reply:
indeed, some variable could be renamed too